### ISSUE TITLE
Improve CLI output for invalid input and general errors

### ISSUE THREAD

**Issue by manuel2258** [heinezen/thunderdome-api#18]
## Context

The current logging output, is rather uninformative, especially if errors occur. 

Specifying a wrong thunderdome token for example looks like this: 

```
INFO:root:Fetching plans for battle ea65..
ERROR:root:Failed to fetch battle ea65...
INFO:root:Transferring points to GitLab...
```

And even a successful fetch could be more informative, as it currently looks like this:

```
INFO:root:Fetching plans for battle ea6...98dacc1...
INFO:root:Transferring points to GitLab...
WARNING:root:Skipping plan 3f...1: No points set for plan
INFO:root:Skipping ra...0: Issue already has a weight set
...
INFO:root:Set weight to 3 for r...0
```

## Wanted solution

It would be nice to state when:
- A thunderdome token is invalid
- A gitlab token is invalid
- A thunderdome game ID is invalid
- Any error happens and the transfer didn't finish successfully
- The transfer fully finished successfully (maybe including a summary how many items were unrated, rated and transfered ...)

Additionally, it would be nice to have a more enduser friendly log formatting, which does not include the INFO:root prefix and maybe some colors?

Thanks for working on this, if you need help or are running out of time I can also create a PR for this!

**Comment by heinezen** [comment]
Yeah, that is probably a good idea. The current log messages were just for debugging and I didn't pay that much attention to usability yet. There are also some quirks of the Thunderdome API that can make error handling difficult.

I'm currently not planning to work on the script that much, since other projects of mine could also use my attention. I just wanted to get the major features done that I like to use for work :D So if you have some time, feel free to make a PR. Otherwise, I'll revisit it some time next month.
